server/odc-migrate/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>com.oceanbase</groupId>
             <artifactId>odc-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>task-plugin-api</artifactId>
```

Comment:
why we need this dependency in the migrate module?

Reply:
cause that I need several pojo the defined by task-plugin-api, such as TimeIncreaseGeneratorConfig and so on~